lib/WTSI/NPG/iRODS/Path.pm

```diff
@@ -294,6 +294,22 @@ sub expected_groups {
     push @groups, $group;
   }
 
+  # For nonconsented split-out human data add a special group.
+  # Give preference to the 'alignment_filter' metadata attribute.
+  # If this attrbute is not defined, examine the path of the object.
+  if (@groups == 1) { # Do not give access to nc human data to multiple groups.
+    my $give_human_subset_access = 0;
+    my @af_avus = $self->find_in_metadata($ALIGNMENT_FILTER);
+    if (not @af_avus and $self->str =~ /_human[.]/xms) {
```

Comment:
Can you expand on why the path fallback check is needed? As this is in the base class the behaviour will be inherited broadly, including by legacy genotype code, which could be given incorrent permissions as a result.

Reply:
I'll mark this as approved, but this change must not break https://github.com/wtsi-npg/genotyping which also depends on this package.